db.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "id": "1",
+    "nom": "Farine",
```

Comment:
can we have labels in English?

- nom => name
- quantite => quantity

Reply:
ok